101-sql-vm-new-storage-ultrassd/README.md

```diff
@@ -3,7 +3,8 @@
 
 Before deploying the template you must have the following
 
-1. **Virtual Network** and **Subnet** a Virtual Network within the same resource group and a subnet must exist in the same region as the VM
+1. **Virtual Network** and **Subnet** Virtual Network and Subnet in same location
```

Comment:
remove the duplicate reference to the vnet/subnet, e.g.

1. **Virtual Network** and **Subnet** in same location

Reply:
Removed duplicate